### ISSUE TITLE
Static content CDN

### ISSUE THREAD

**Issue by ashleyhull-versent** [pixelfed/pixelfed#3782]
Does Pixelfed have similar to the mastodon CDN_HOST? to serve the static content?

**Comment by mitexleo** [comment]
No 👎... Not seeing such variable on Docs

**Comment by atlza** [comment]
I was thinking about this, this is really needed. Especially when working with S3.

**Comment by foozmeat** [comment]
Agreed, we definitely need this.

**Comment by luxiaba** [comment]
Agreed, we need this, btw the mang variables on docs in the docs are not very clear.

**Comment by Simon-Laux** [comment]
let's close this in favor of #5656 (because that has a workaround and a more detailed description)
